includes/API/GetPageAllRevsHandler.php

```diff
@@ -2,38 +2,37 @@
 
 namespace DataAccounting\API;
 
+use DataAccounting\get_page_all_revs;
 use MediaWiki\Rest\SimpleHandler;
 use Wikimedia\ParamValidator\ParamValidator;
-use MediaWiki\MediaWikiServices;
-
-use DataAccounting\get_page_all_revs;
 
 # include / exclude for debugging
-error_reporting(E_ALL);
-ini_set("display_errors", 1);
+error_reporting( E_ALL );
+ini_set( "display_errors", 1 );
 
-require_once(__DIR__ . "/../ApiUtil.php");
+require_once( __DIR__ . "/../ApiUtil.php" );
 
 class GetPageAllRevsHandler extends SimpleHandler {
-    /** @inheritDoc */
-    public function run( $page_title ) {
-        #Expects Page Title and returns ALL verified revisions
-        return \DataAccounting\get_page_all_revs($page_title);
-    }
-
-    /** @inheritDoc */
-    public function needsWriteAccess() {
-        return false;
-    }
-
-    /** @inheritDoc */
-    public function getParamSettings() {
-        return [
-            'page_title' => [
+
+	/** @inheritDoc */
+	public function run( $page_title ) {
+		#Expects Page Title and returns ALL verified revisions
+		return get_page_all_revs( $page_title );
```

Comment:
@JeroenDeDauw this is the cause of the error. I don't think this is done by an autoformatter. There is in fact a namespace declaration in `ApiUtil`, and it's `DataAccounting` instead of `DataAccounting\API`. Some of the functions in that file are actually used outside of the API stuff, e.g. `getWitnessData`. I think it is confusing to put them in ApiUtil.php. They should live in Util.php instead.

I will wait for your response on whether to remove the namespace declaration in ApiUtil.php or not.

Reply:
This was the formatting alright. I did not make any other changes as part of that commit.

Note the `use DataAccounting\get_page_all_revs;` import at the top of the file. The formatting moved it but it was already there. I think this is why it dropped the namespace over here, since it though it was already imported.

You should be able to fix the import at the top of the file to resolve the issue

Reply:
> You should be able to fix the import at the top of the file to resolve the issue

Did you mean `use DataAccounting\get_page_all_revs as get_page_all_revs;`?

Reply:
Yes

Reply:
@JeroenDeDauw I think regardless of whether what I did is a bad practice or not, the auto-formatter shouldn't introduce a change in the code logic. Should this be filed as a bug upstream?